examples/Tutorial.ipynb

```diff
@@ -519,46 +518,46 @@
       "Objective Function Score (Test) = 1.0\n",
       "\n",
       "Fold: 4\n",
-      "Model Parameters: {\"C\": 100.0, \"class_weight\": null, \"dual\": false, \"fit_intercept\": true, \"intercept_scaling\": 1, \"l1_ratio\": null, \"max_iter\": 1000, \"multi_class\": \"auto\", \"n_jobs\": null, \"penalty\": \"l2\", \"random_state\": 123456789, \"solver\": \"liblinear\", \"tol\": 0.0001, \"verbose\": 0, \"warm_start\": false}\n",
-      "Grid Objective Score (Train) = 0.9770114942528737\n",
+      "Model Parameters: {\"C\": 10.0, \"class_weight\": null, \"dual\": false, \"fit_intercept\": true, \"intercept_scaling\": 1, \"l1_ratio\": null, \"max_iter\": 1000, \"multi_class\": \"auto\", \"n_jobs\": null, \"penalty\": \"l2\", \"random_state\": 123456789, \"solver\": \"liblinear\", \"tol\": 0.0001, \"verbose\": 0, \"warm_start\": false}\n",
+      "Grid Objective Score (Train) = 0.9555555555555556\n",
       "+------------+----------+--------------+-------------+-------------+----------+-------------+\n",
       "|            |   setosa |   versicolor |   virginica |   Precision |   Recall |   F-measure |\n",
       "+============+==========+==============+=============+=============+==========+=============+\n",
       "|     setosa |      [3] |            0 |           0 |       1.000 |    1.000 |       1.000 |\n",
       "+------------+----------+--------------+-------------+-------------+----------+-------------+\n",
-      "| versicolor |        0 |          [2] |           2 |       1.000 |    0.500 |       0.667 |\n",
+      "| versicolor |        0 |          [3] |           0 |       1.000 |    1.000 |       1.000 |\n",
       "+------------+----------+--------------+-------------+-------------+----------+-------------+\n",
-      "|  virginica |        0 |            0 |         [4] |       0.667 |    1.000 |       0.800 |\n",
+      "|  virginica |        0 |            0 |         [4] |       1.000 |    1.000 |       1.000 |\n",
       "+------------+----------+--------------+-------------+-------------+----------+-------------+\n",
       "(row = reference; column = predicted)\n",
-      "Accuracy = 0.8181818181818182\n",
-      "Objective Function Score (Test) = 0.8181818181818182\n",
+      "Accuracy = 1.0\n",
```

Comment:
i do not see any update in the dataset used, but the results differ a lot. did i miss something?